batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/PersonDao.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.entity.Person;
+import com.lftechnology.batch7crud.utils.DbConnection;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.naming.InitialContext'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)